package.json

```diff
@@ -46,12 +46,13 @@
     ],
     "testPathIgnorePatterns": [
       "/node_modules/",
+      "<rootDir>/integration-tests/create-react-app",
       "<rootDir>/packages/react-percy-webpack/src/entry/__tests__/test-project/"
     ]
   },
   "lint-staged": {
     "*.js": [
-      "eslint --fix --color --no-ignore",
```

Comment:
Ahh gotcha, makes sense. I'm not in front of my computer just now but I can try in the morning, but did you try using the --ignore-pattern flag instead, as mentioned here (under the fourth section) https://github.com/eslint/eslint/pull/6844#issuecomment-239597082? It looks like that would allow lint staged to run on dotfiles without having to completely disregard the other eslint ignore rules.